packages/vue2/src/components/Root.vue

```diff
@@ -515,7 +515,11 @@ export default {
     },
 
     refresh(payload) {
-      this.getData(payload);
+      if (this.paginationMode == "infinite") {
+        this.setPage(1);
```

Comment:
Need to add payload support in the setPage method.
Provide `payload` as second argument.